src/components/3d_editor/advanced_geometry/python_transformation/MaterialsSelector.tsx

```diff
@@ -0,0 +1,62 @@
+import CheckBox from "@mui/icons-material/CheckBox";
+import CheckBoxOutlineBlank from "@mui/icons-material/CheckBoxOutlineBlank";
+import Autocomplete from "@mui/material/Autocomplete";
+import Checkbox from "@mui/material/Checkbox";
+import Chip from "@mui/material/Chip";
+import TextField from "@mui/material/TextField";
+import React from "react";
+
+interface MaterialsSelectorProps {
```

Comment:
Let's clean up the linter messages in this PR, pls
![Screenshot 2023-11-24 at 12 42 03 PM](https://github.com/Exabyte-io/materials-designer/assets/721112/89e2de1d-6af8-446d-8807-38956af9db60)

Reply:
That's why I used ts-ignore previously. 
This will be solved correctly when Made.js is moved to TS and type for Material exists.